hasher-matcher-actioner/.dockerignore

```diff
@@ -0,0 +1,14 @@
+webapp/
```

Comment:
nit: ELI5 why we want this (preferably inline). the set of dot/config files for hma has grown to the point where it might be work an appendix in CONTRIBUTING.md to keep track